.github/workflows/go.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.18
+          go-version: 1.19
```

Comment:
apologies for the drive by :)

may be worth setting `go-version-file: 'path/to/go.mod'` rather than manually keeping in line https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file

Reply:
ohhhhhhhhh they support that 😻  definitely yes !

Reply:
looks nice thanks @jbpratt

Reply:
gotta checkout code first it looks like @vdemeester

Reply:
:) okay i'll go away :heart:

Reply:
> :) okay i'll go away heart

yeah I figured that out after 😝